lib/inferno/dsl/fhir_validation.rb

```diff
@@ -1,63 +1,143 @@
 module Inferno
   module DSL
+    # This module contains the methods needed to configure a validator to
+    # perform validation of FHIR resources. The actual validation is performed
+    # by an external FHIR validation service. Tests will typically rely on
+    # `assert_valid_resource` for validation rather than directly calling
+    # methods on a validator.
+    #
+    # @example
+    #
+    #   validator do
+    #     url 'http://example.com/validator'
```

Comment:
I think putting the URL in at this level will pretty tightly bind the addresses of the various services to the definition of the test, which seems a bit awkward to me.  It would be like defining `postgres://postgres_host:4543` in your models files, instead of in a separate configuration area.  Maybe in this section you'd define constraints on which validators are acceptable (e.g. hl7-validator, or ruby-validator... along with versions), and exclusions/additional validations... and then in `config/validators.rb` provide the list of validators that will be queried on loadup... and dynamically matched up with suites (and if there are no matches, provide some way in the API/console to convey that issue).  This would be really nice for our `inferno_deployment` repo, where it wouldn't have to make sure it happened to expose all the right services at the right spots that various tests said.  

I'm ok going with the approach in this PR to start but maybe we can get some consensus on whether or not we should make additional tickets to move the URL out of here prior to merging, while we are on the topic?

Reply:
It can be set via an environment variable.